evm/deploy-contract.js

```diff
@@ -340,25 +386,11 @@ const program = new Command();
 
 program.name('deploy-contract').description('Deploy contracts using create, create2, or create3');
 
-program.addOption(
-    new Option('-e, --env <env>', 'environment')
-        .choices(['local', 'devnet', 'stagenet', 'testnet', 'mainnet'])
-        .default('testnet')
-        .makeOptionMandatory(true)
-        .env('ENV'),
-);
-program.addOption(new Option('-a, --artifactPath <artifactPath>', 'artifact path').makeOptionMandatory(true));
-program.addOption(new Option('-c, --contractName <contractName>', 'contract name').makeOptionMandatory(true));
-program.addOption(new Option('-n, --chainNames <chainNames>', 'chain names').makeOptionMandatory(true).env('CHAINS'));
-program.addOption(new Option('--skipChains <skipChains>', 'chains to skip over'));
+addDeploymentOptions(program, true, true, true, true, true);
```

Comment:
using flags like this is quite unreadable. Use an object instead with named fields. If the field is set, say skipPrivateKey, then it'll skip it, otherwise it'll include by default